systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -917,10 +917,10 @@
     
     function body_id = findLinkId(model,linkname,robot,error_level)
       % @param robot can be the robot number or the name of a robot
-      % robot=0 means look at all robots
+      % robot<0 means look at all robots
       % @param error_level >0 for throw error, 0 for throw warning, <0 for do nothing. @default throw error
       % @ingroup Kinematic Tree
-      if nargin<3 || isempty(robot), robot=0; end
+      if nargin<3 || isempty(robot), robot=-1; end
```

Comment:
i had checked all of these, but had not thought about the chaos that would ensue if -1 was passed in.  they are handled now.  good catch.